cmd/run.go

```diff
@@ -407,6 +440,22 @@ func (c *cmdRun) flagSet() *pflag.FlagSet {
 	return flags
 }
 
+func (c *cmdRun) setupTracerProvider(test *loadedAndConfiguredTest) error {
+	ro := test.preInitState.RuntimeOptions
+	if ro.TracesOutput.String == "none" {
+		test.preInitState.TracerProvider = trace.NewNoopTracerProvider()
+		return nil
+	}
+
+	tp, err := trace.TracerProviderFromConfigLine(c.gs.Ctx, ro.TracesOutput.String)
```

Comment:
nit: you can provide the `globalcCtx` as is done in the cancelation timeout

Reply:
Right, that makes total sense. Fixed in [663c6c2](https://github.com/grafana/k6/pull/3445/commits/663c6c2e9d2ad597f4ca419107e4783dc465507d).